### ISSUE TITLE
Cannot require concurrent/map on JRuby

### ISSUE THREAD

**Issue by mperham** [ruby-concurrency/concurrent-ruby#439]
Same code works on MRI.

```
NameError: uninitialized constant Concurrent::Collection::JRubyMapBackend
        const_missing at org/jruby/RubyModule.java:3144
  <module:Collection> at /Users/mike/.gem/jruby/2.2.2/gems/concurrent-ruby-1.0.0.pre3-java/lib/concurrent/map.rb:14
  <module:Concurrent> at /Users/mike/.gem/jruby/2.2.2/gems/concurrent-ruby-1.0.0.pre3-java/lib/concurrent/map.rb:6
                <top> at /Users/mike/.gem/jruby/2.2.2/gems/concurrent-ruby-1.0.0.pre3-java/lib/concurrent/map.rb:4
              require at org/jruby/RubyKernel.java:940
                <top> at /Users/mike/src/sidekiq/lib/sidekiq/processor.rb:1
```

**Comment by mperham** [comment]
It worked when I did `require 'concurrent'` per the docs.

**Comment by jdantonio** [comment]
This is probably related to #434, but I will verify tonight.

**Comment by jdantonio** [comment]
This should be addressed in the 1.0.0.pre4 release which I just uploaded to GitHub. Most of our classes require requiring the entire gem (something we plan to change in 2.0), but the three classes imported from thread_safe (Array, Map, and Hash) should work fine by requiring just that class.

Please let me know if this works so I can close the issue.

**Comment by mperham** [comment]
pre4 works great, thanks.
